### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by franclobo**
### Vet clinic database: create animals table.
Find all animals whose name ends in "mon".

namedMON

List the name of all animals born between 2016 and 2019.

dateOfBirth

List the name of all animals that are neutered and have less than 3 escape attempts.

escape

List the date of birth of all animals named either "Agumon" or "Pikachu".

birth

List name and escape attempts of animals that weigh more than 10.5kg

weight

Find all animals that are neutered.

neutered

Find all animals not named Gabumon.

NoGabumon

Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 10.4kg or 17.3kg)

WeightOr

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+'Gabumon'::text, '15/11/2018'::date, '2'::integer, true::boolean, '8'::numeric)
+returning id;
+INSERT INTO public.animals (
+name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (
+'Devimon'::text, '12/05/2017'::date, '5'::integer, true::boolean, '10.23'::numeric)
+returning id;
+INSERT INTO public.animals (
+name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (
+'Agumon'::text, '03/02/2020'::date, '0'::integer, true::boolean, '10.23'::numeric)
+returning id;
```
Comment (sja-thedude) [inline data.sql]: - [x] Kindly add Pikachu animal data, you have repeated Agumon data. Fix it.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @franclobo, This is @sja-thedude your code reviewer.

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Correct Gitflow

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (franclobo) [reply data.sql]: Thank you @sja-thedude I have already fixed it.

**Review by zhadier — APPROVED** [review]
## Hi @franclobo  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 



# Congratulations! 🎉 

✔️ Awesome Queries 

### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**